src/viam/services/motion/client.py

```diff
@@ -131,6 +141,51 @@ async def move_on_globe(
         response: MoveOnGlobeResponse = await self.client.MoveOnGlobe(request, timeout=timeout)
         return response.success
 
+    async def move_on_globe_new(
+        self,
+        component_name: ResourceName,
+        destination: GeoPoint,
+        movement_sensor_name: ResourceName,
+        obstacles: Optional[Sequence[GeoObstacle]] = None,
+        heading: Optional[float] = None,
+        configuration: Optional[MotionConfiguration] = None,
+        *,
+        extra: Optional[Mapping[str, ValueTypes]] = None,
+        timeout: Optional[float] = None,
+    ) -> str:
+        """
+        **Experimental**: use move_on_globe instead.
+        Move a component to a specific latitude and longitude, using a ``MovementSensor`` to check the location.
+
+        Note: Is non blocking.
+
+        Args:
+            component_name (ResourceName): The component to move
+            destination (GeoPoint): The destination point
+            movement_sensor_name (ResourceName): The ``MovementSensor`` which will be used to check robot location
+            obstacles (Optional[Sequence[GeoObstacle]], optional): Obstacles to be considered for motion planning. Defaults to None.
+            heading (Optional[float], optional): Compass heading to achieve at the destination, in degrees [0-360). Defaults to None.
```

Comment:
nit: `in degrees [0-360)` ends with `)` instead of `]`.

Reply:
This might be on purpose -- I assume it's set notation? 0 is inclusive, 360 is not?

Reply:
nope, @purplenicole730 was right, was just a typo